Versionsmigration/4.1-5.4/xplangml-migration-41-54.halex.alignment.xml

```diff
@@ -10228,12 +10228,12 @@ c.text['SO'].values()[0].each { tex -&gt;&#xD;
         </target>
         <complexParameter name="script">
             <core:text xmlns:core="http://www.esdi-humboldt.eu/hale/core" xml:space="preserve">
-def c = _.context.collector()&#xD;
-&#xD;
-if (dientZurDarstellungVon.p.first()) {&#xD;
-	dientZurDarstellungVon.p.each { dar -&gt;&#xD;
-		c.dars[dar.p.href.value().toString().replaceAll('#','')] &lt;&lt; id.p.value()&#xD;
-	}&#xD;
```

Comment:
In order to make commits easier to review, I would suggest to do one commit with all the line ending changes (changed with hale studio version 5.0.0 (?)) and one with the actual changes.
Is not necessary to change it now for this small commit that can still be reviewed but for future cases.